moesif-servlet/pom.xml

```diff
@@ -161,7 +161,7 @@
             <id>sign-artifacts</id>
             <phase>verify</phase>
             <goals>
-              <goal>sign</goal>
+<!--              <goal>sign</goal>-->
```

Comment:
@bakennedy why was this commented?

Reply:
This is a mistaken commit addition, thanks for catching it. Will revert